Dockerfile.sql_cli

```diff
@@ -0,0 +1,9 @@
+ARG PYTHON_MAJOR_MINOR_VERSION="3.9"
+# ARG PYTHON_BASE_IMAGE="python:${PYTHON_MAJOR_MINOR_VERSION}-slim-bullseye"
+ARG PYTHON_BASE_IMAGE="python:${PYTHON_MAJOR_MINOR_VERSION}"
```

Comment:
Why commented code?

Reply:
I used a non-slim image so that it could have required libraries that are needed for python-sdk installation. But, figured out now required gcc installation and the slim base image works now. Removed the comment.